src/sass/ui/_minimal.scss

```diff
@@ -0,0 +1,26 @@
+.user-interface {
+    &--minimal {
```

Comment:
Any reason for picking a double-dash as separator? At first I mis-interpreted this as a [custom CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/--*).

Reply:
Yeah thats my bad. I am used to work with CSS BEM (http://getbem.com/introduction/) and there it is required to do such things with double dashes or / and double underlines. 

But as I see, this project doesn't use this standard so I will change it to a single dash :)

Reply:
No worries, I simply wasn't aware of BEM :)